octopoes/octopoes/repositories/ooi_repository.py

```diff
@@ -214,19 +214,17 @@ def commit(self):
     @classmethod
     def serialize(cls, ooi: OOI) -> dict[str, Any]:
         # export model with pydantic serializers
-        export = json.loads(ooi.model_dump_json())
+        export = json.loads(ooi.json())
```

Comment:
The `json` method is deprecated; use `model_dump_json` instead

Reply:
LOL that was my PR I reverted.

Reply:
We actually have a lot of them in the code